analysis/common_variables.py

```diff
@@ -265,6 +265,27 @@ def generate_common_variables(index_date_variable, exposure_end_date_variable, o
         "tmp_out_date_vascular_dementia_snomed", "tmp_out_date_vascular_dementia_hes", "tmp_out_date_vascular_dementia_death",
     ),
 
+    # Lewy body
+    # Primary
+    tmp_out_date_lewy_body_dementia_snomed=patients.with_these_clinical_events(
+        lewy_body_snomed,
+        returning="date",
+        between=[f"{index_date_variable}",f"{outcome_end_date_variable}"],
+        date_format="YYYY-MM-DD",
+        find_first_match_in_period=True,
+        return_expectations={
+            "date": {"earliest": study_dates["pandemic_start"], "latest" : "today"},
+            "rate": "uniform",
+            "incidence": 0.1,
+        },
+    ),
+    # HES NO ICD10
+    # ONS NO ICD10
+    # Combined lewy body disease
+    out_date_lewy_body_dementia = patients.minimum_of(
+        "tmp_out_date_lewy_body_dementia_snomed", 
+    ),
+
     # Other dementias
```

Comment:
The other dementia code list should also be updated to remove the lewy body dementia codes as these definitions currently overlap.

Reply:
Code list updated and added.

Reply:
The other dementia code list in the Excel document contains 132 codes, while [this list](https://www.opencodelists.org/codelist/bristol/other-dementias-snomed-ct-v13/494d586a/#full-list) contains 136 codes. Please can you remove the four additional codes?